samples/core/guide/autograph.ipynb

```diff
@@ -609,6 +646,217 @@
       "execution_count": 0,
       "outputs": []
     },
+    {
+      "metadata": {
+        "id": "XY4UspHmZNdL",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## Interoperation with `tf.Keras`\n",
+        "\n",
+        "Now that you've seen the basics, let's build some real model components with autograph.\n",
+        "\n",
+        "It's relatively simple to integrate `autograph` with `tf.keras`. But remember that batchng is essential for performance. So the best candidate code for conversion to autograph is code where the control flow is decided at the _batch_ level. If decisions are made at the individual _example_ level you will still need to index and batch your examples to maintain performance while appling the control flow logic. \n",
```

Comment:
Edit:
It's easy to use `autograph` with `tf.keras`, but batching is essential for performance. The best code to convert to AutoGraph is code where the control flow is decided at the *batch* level. If making decisions at the individual *example* level, you must index and batch the examples to maintain performance while applying the control flow logic.

Reply:
I suggest keeping the observation about batching as a separate callout, e.g. in a separate paragraph with "Tip: remember that the inputs are usually batched examples, ...". The guidance is otherwise not strongly related to the keras interop.

Reply:
> Edit:

Done. 

> not strongly related to the keras interop.

Moved up to the of examples.